crates/sqlbuiltins/src/functions/scalars/postgres.rs

```diff
@@ -1,10 +1,10 @@
 use datafusion::logical_expr::expr::ScalarFunction;
 use pgrepr::compatible::server_version_with_build_info;
 
+use super::df_scalars::array_to_string;
+use super::*;
```

Comment:
I thought we wanted to forbid usage of `super::*`

Reply:
we do, but rustfmt doesn't support that. 

I will see if there's a clippy rule or something that we can use.

Reply:
https://rust-lang.github.io/rust-clippy/rust-1.56.0/index.html <- looks like there's something. will do that separately.